base/reflection.jl

```diff
@@ -941,13 +941,41 @@ function method_exists(f::ANY, t::ANY)
         typemax(UInt)) != 0
 end
 
+"""
+    isambiguous(m1, m2, [allow_bottom_tparams=true]) -> Bool
```

Comment:
unrelated aside, but I think we should gradually aim to get away from using square brackets to denote optional arguments and uniformly use the julia syntax convention for them in docstrings